src/elements/common/header/Header.scss

```diff
@@ -16,10 +16,4 @@
         flex: 1;
         padding-left: 20px;
     }
-
-    input[type='search'] {
-        width: 100%;
-        font: inherit;
-        -webkit-appearance: none;
-    }
```

Comment:
I'm not too sure what these lines were intended for. my assumption is this was related to a Safari issue but we don't target `input[type='search']` for any other components so I don't know why this was needed.

I think it's okay to remove the block now:

- `width: 100%;` is being added here: https://github.com/box/box-ui-elements/blob/8c0e2989690b882b1ce299566ac0105d576ad1ff/src/elements/common/base.scss#L25
- `font: inherit;` is applied here: https://github.com/box/box-ui-elements/blob/8c0e2989690b882b1ce299566ac0105d576ad1ff/src/elements/common/base.scss#L62
- `-webkit-appearance: none;` from my testing this wasn't having an impact in most recent version of browsers. I also don't think we want this based on this: https://github.com/box/box-ui-elements/blob/b9d196fcf330d38de87890ca8ccf4a5c8433d313/src/styles/common/_forms.scss#L106